### PR TITLE
Code review to merge master

### PR DESCRIPTION
**Pull request by c-m-a**
We the YouTube Clone player page and we think we have finished the job!

### CODE REVIEW

**Review by Forison — COMMENTED** [review]
# Pull request under review by ADDO

**Review by Forison — CHANGES_REQUESTED** [review]
Hi @Chincoya   and @anicma 
I would like to welcome you once again in the program.👏👏👏
# I would have loved to make you proceed to the next project but unfortunately, I can not until you rectify some issues I have found with your project.😇😇😇
* Friendly reminder: You should use [Github flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in this project. However, changing it now would take too much time. Please make sure you use [Github Flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in the next project.*🙂🙂🙂

when submitting a pull request make sure to create a pull request and submit the path of that pull request.

- #### **[REQUIRED]** README 🙂 
you need to add a readme file to your project.
Your readme file is supposed to be descriptive enough(readme has to describe the project you have worked on in detail so that anyone who takes your project will have a fair view of what the project is about when they view the readme).
- ### **[REQUIRED]** LINTERS🙂
you did not configure **linters** as required. I suggest you read this [article](https://github.com/microverseinc/linters-config#linters-config).

- ### **[REQUIRED]** HTML🙂
1. I have taken notice of several violations of the HTML semantics in your project. Please read this [article](https://www.w3schools.com/html/html5_semantic_elements.asp) for clarification on the Topic.

2. You should fix the errors given to you by the[ validator ](https://validator.w3.org)when uploading the index file.🙂

- ### **[REQUIRED]** CSS 🙂

Do not use id as selectors in your project(Id selectors have priority over other kinds of selectors. This can make it harder to add new rules using less specific selectors, leading to specificity wars), use class instead. read on the best practices [here](https://speckyboy.com/good-bad-css-practices/)

## **[REQUIRED]** MAIN PROJECT IMPLEMENTATION FLAWS
I have made the following observations about your project:

- Your project looks good but there has to be a gap between ```.player-col``` and ```. thumbnail```.Currently the two overlap. try to use semantic HTML here you can have  ```<main>```  and ```<aside>```

Guys, please fix the errors I have pointed out and submitted your project for another feel free to contact me on [LinkedIn](https://www.linkedin.com/feed/), [Twitter](https://twitter.com/ADDOBOAKYE1) or slack for further clarification if necessary. I would be happy to assist you.
💪💪💪 happy coding 💪💪💪

**Comment by c-m-a** [comment]
Hi @Forison, we fixed all the requirements that you suggested to us! Let us know if you see anything to improve! :)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.2/css/all.css">
+    <title>YouClone | Player: How to build a YouTube Clone</title>
+
+</head>
+<body>
```
Comment (samgaco) [inline index.html:12]: It'd be good if you add `<main>` tags for the dominant content of your body moving on

**Review by samgaco — APPROVED** [review]
Hi @anicma and @Chincoya , amazing project you have worked on :100:  one of the best I have seen. It's approved :white_check_mark: hope you keep this kind good work up :arrow_up: 

You may merge to the master branch and submit the project completion form.

--------------------

For your next projects have this into consideration :arrow_down: 

* Friendly reminder: You should use [Github flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in this project. You are now merging from a development branch, ideally, you would merge from a feature branch (the name of the branch should define one characteristic you are working on for the project, please watch the video in the Github flow section for further clarifications. [Github Flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in the next project. 

---------------------------
* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

File: index.html (hunk 1 above)
Comment (c-m-a) [reply index.html:12]: Thanks for your feedback! We are going to use this from now on! :)

**Pull request merged**